librz/core/cconfig.c

```diff
@@ -2651,6 +2651,21 @@ static bool cb_search_in(void *user, void *data) {
 	return false;
 }
 
+static bool cb_search_show_progress(void *user, void *data) {
+	RzConfigNode *node = (RzConfigNode *)data;
+	if (rz_str_is_true(node->value)) {
+		free(node->value);
+		node->value = rz_str_dup("true");
+	} else if (rz_str_is_false(node->value)) {
+		free(node->value);
+		node->value = rz_str_dup("false");
+	} else if (strcmp(node->value, "intervals")) {
```

Comment:
```suggestion
	} else if (!RZ_STR_EQ(node->value, "intervals")) {
```

Reply:
Also, the the `intervals` case is missing?

Reply:
Nothing needs to be done for the `intervals` case and the code can just return true.